x/compute/internal/keeper/keeper.go

```diff
@@ -89,8 +91,41 @@ func (k Keeper) Create(ctx sdk.Context, creator sdk.AccAddress, wasmCode []byte,
 	return codeID, nil
 }
 
+// GetSignBytes returns the signBytes of the tx for a given signer
+// This is a copy of cosmos-sdk function (cosmos-sdk/x/auth/types/StdTx.GetSignBytes(), because the original takes a wrong sequence number
+func GetSignBytes(ctx sdk.Context, acc exported.Account, tx auth.StdTx) []byte {
```

Comment:
If this is because of an issue, can you also link to it?

Reply:
It's not.
This is because the original `GetSignBytes` was probably meant to be used before the transaction gets processed, and the sequence that gets returned is an increment of what we need.
This is why we use `acc.GetSequence() - 1`

Reply:
Ahh, I see. that makes sense

Reply:
can you maybe make the comment clearer?